src/services/API/sqs.js

```diff
@@ -0,0 +1,50 @@
+import { SQSClient, SendMessageCommand } from "@aws-sdk/client-sqs";
+
+const REGION = "ap-south-1";
+const queueURL = "https://sqs.ap-south-1.amazonaws.com/111766607077/EventQueue";
+const sqsClient = new SQSClient({
+  region: REGION,
+  credentials: {
+    accessKeyId: 'AKIARUBOPCTSSU3W7SML',
+    secretAccessKey: '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',
+  },
+});
+
+async function sendSQSMessage(purpose, purposeParams, redirectTo, redirectID, userID, doesUserExist) {
+    const messageBody = [
+      {
+        DateTime: Date.now().toString(),
+        Purpose: {
+          type: purpose,
+          subtype: purposeParams,
+          params: {
+            platform: redirectTo,
+            id: redirectID,
+          },
+        },
+        auth_type: "SRN", //for now, this is hardcoded
```

Comment:
Or you can send this information as a param into the `sendSQSMessage` function, and where you're calling this function, you can pass "SRN" there